.circleci/config.yml

```diff
@@ -1,14 +1,30 @@
----
-version: 2
+version: 2.1
 
 references:
   images:
     middleman: &MIDDLEMAN_IMAGE hashicorp/middleman-hashicorp:0.3.44
+    ubuntu: &UBUNTU_IMAGE ubuntu-1604:201903-01
 
   cache:
     rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "content/Gemfile.lock" }}
 
 jobs:
+  website-test:
+    machine:
+      image: *UBUNTU_IMAGE
+    steps:
+      - checkout
+
+      - run:
+          name: gem and bundle install 
+          command: |
+            gem install bundler -v '1.17.3' --no-document
```

Comment:
Just as fyi/context, the need to specify this bundler version was originally an artifact of Travis's build workers being configured in some bizarre way. I don't know whether Circle's ubuntu image would have the same problem, but it should be harmless to keep this, and the experiments we'd need to do before removing it are probably not a great use of our time, given that our middleman platform is on borrowed time anyway.